babel.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
Can we move this and the jest config to the `config` directory?

Reply:
I moved jest config files, but for now, babel.config should stay at root level until I figure out how to deal with it (I couldn't find any clue how to specify babel path for jest)